.github/workflows/linting_skipper.yml

```diff
@@ -10,6 +10,12 @@ on:
       - "**/pyproject.toml"
 
 jobs:
+  license-header:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Skip mypy
+        run: echo "Skipped mypy"
```

Comment:
unrelated: can we also update `paths-ignore`? they seem outdated

Reply:
Do we need them at all?

Reply:
Ah, I don't know :slightly_smiling_face: